src/fidesops/tasks/__init__.py

```diff
@@ -1,25 +1,37 @@
+from typing import Any, Dict, MutableMapping
+
 from celery import Celery
 from celery.utils.log import get_task_logger
+from fideslib.core.config import load_toml
 
 from fidesops.core.config import config
+from fidesops.util.logger import NotPii
 
 logger = get_task_logger(__name__)
 
 
-def _create_celery() -> Celery:
+def _create_celery(config_path: str = config.execution.CELERY_CONFIG_PATH) -> Celery:
     """
     Returns a configured version of the Celery application
     """
     logger.info("Creating Celery app...")
     app = Celery(__name__)
 
-    broker_url = config.execution.CELERY_BROKER_URL or config.redis.CONNECTION_URL
-    app.conf.update(broker_url=broker_url)
+    celery_config: Dict[str, Any] = {
+        # Defaults for the celery config
```

Comment:
Is this comment correct? It says celery config, but then uses the redis config.

Reply:
This is intentional, we'd like Celery to default to using the Redis instance that the Fidesops execution uses.